### PR TITLE
Blog - Initial blog setup

### PR DESCRIPTION
**Pull request by Emmy-github-webdev**
## Blog - Initial blog setup

In this project I
- Create a new app called blog
- Generate blog controller, model, and view with "Rails generate resources Blog title: string text: text"
- Set up Postgres database with the project
- Generate sample text with seeds
- Setup root route for blog

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
```
Comment (SajjadAhmad14) [inline config/database.yml]: The Database adapter should be using `Postgresql` not `sqlite`. Please use `Postgres` as a database.

File: app/controllers/blogs_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class BlogsController < ApplicationController
+  def index
```
Comment (SajjadAhmad14) [inline app/controllers/blogs_controller.rb:2]: Please add `users_controller` and `posts_controller` with #index and #show actions. Moreover, please add corresponding view files.

File: config/routes.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  root to: 'blogs#index'
+  resources :blogs
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```
Comment (SajjadAhmad14) [inline config/routes.rb]: Please add users and posts routes  for `index` and `show` actions to `config/routes.rb`.

File: config/routes.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+  # Defines the root path route ("/")
```
Comment (SajjadAhmad14) [inline config/routes.rb:11]: Please go through project [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md) and make sure every requirement is met.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Emmanuel, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: config/database.yml (hunk 1 above)
Comment (Emmy-github-webdev) [reply config/database.yml]: Database set to Postgresql

File: app/controllers/blogs_controller.rb (hunk 2 above)
Comment (Emmy-github-webdev) [reply app/controllers/blogs_controller.rb:2]: user_controller, blog_controller added with index and show methods respectively

File: config/routes.rb (hunk 3 above)
Comment (Emmy-github-webdev) [reply config/routes.rb]: Users and posts routes for index and show

File: config/routes.rb (hunk 4 above)
Comment (Emmy-github-webdev) [reply config/routes.rb:11]: Root root added

**Review by addod19 — APPROVED** [review]
### Hi @Emmy-github-webdev, 

Your project is complete!  Nice job addressing the comments by the previous code reviewer 👍 There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Emmy-github-webdev** [comment]
Thank you @addod19

**Pull request closed without merging**